android/library/src/main/java/com/liferay/mobile/screens/gallery/interactor/upload/GalleryUploadService.java

```diff
@@ -29,17 +32,39 @@
  */
 public class GalleryUploadService extends IntentService {
 
+	private boolean shouldCancel = false;
```

Comment:
default values are not usually (in common code conventions) specified explicitely